src/codegen/generators/pou_generator.rs

```diff
@@ -413,14 +484,27 @@ impl<'ink, 'cg> PouGenerator<'ink, 'cg> {
                     m.get_name(),
                     &index.get_associated_type(m.get_type_name())?,
                 );
+
+                if let Some(block) = self.llvm.builder.get_insert_block() {
+                    debug.add_variable_declaration(
```

Comment:
shall we also register variable declaration for the return variable? (aggregate and normal ones?)